scripts/harvard_forests.json

```diff
@@ -11,9 +11,8 @@
         "harvard forest",
         "spatial-data"
     ],
-    "url": "FILL",
     "ref": "http://harvardforest.fas.harvard.edu/",
-    "version": "1.0.2",
+    "version": "2.0.0",
```

Comment:
@harshitbansal05 let's call this `harvard-forest, harvard_forest.json`  from the url http://harvardforest.fas.harvard.edu/